src/main/java/uk/gov/justice/hmpps/prison/api/model/OffenderDamageObligationModel.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.justice.hmpps.prison.api.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Builder;
+import lombok.Getter;
+
+import javax.validation.constraints.NotBlank;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+
+@Builder
+@Getter
+@ApiModel(description = "Damage obligation for an offender")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class OffenderDamageObligationModel {
+
+    @ApiModelProperty(value = "Identifier of damage obligation", example = "1", position = 1)
+    private Long id;
+
+    @ApiModelProperty(value = "Offender number", example = "A12345", position = 2)
```

Comment:
what is this field?  Would be good to put an example in the correct format e.g. G5033UT